packages/components/src/components/Modal/components/Backdrop.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Please get rid of this super nested `components` folder. It fine to have this files in `Modal` folder directly.

Reply:
The super nested `components` folder has been gotten rid of.
https://github.com/trezor/trezor-suite/pull/5132/commits/5f2d98e43c7989356a27ee5737f4541d954eb8c3